tests/parsing/ConfParserTest.py

```diff
@@ -200,3 +200,38 @@ def test_settings_override_warning(self):
                                             'already been defined in section '
                                             'name. The previous setting will '
                                             'be overridden.')
+
+
+class ConfParserTest_(unittest.TestCase):
+    test_file = """
+    [Default]
+    ignore = aaa
```

Comment:
If it's read as a string it will be independent of `,`. Please let me know what's your take on this. I tried comparing it with a list 
```
    [Default]
    ignore = aaa,bbb
    ignore += ccc
```
```
expected = ['aaa', 'bbb', 'ccc']
```
But the assertion through the following error
```
AssertionError: ['aaa', 'bbb', 'ccc'] != <Setting object(key='ignore', value='aaa,[122 chars]5f28> 
```